localsettings/core.txt

```diff
@@ -1,7 +1,8 @@
 // Generic verbose error reporting stuff
+$isCli = PHP_SAPI === 'cli' || PHP_SAPI !== 'phpdbg';
```

Comment:
This looks wrong, compared to code in `wfIsCLI()`.

Reply:
Oops, forgot to invert the second half...